### PR TITLE
Organize storybook categories

### PR DESCRIPTION
**Pull request by stellarhoof**
Now we've got 4 categories:
- Components (Unthemed)
- Search Components (Unthemed)
- Components (Grey Vest)
- Search Components (Grey Vest)

### CODE REVIEW

**Comment by stellarhoof** [comment]
image

File: stories/explorer/index.stories.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import 'babel-polyfill'
 import { storiesOf } from '@storybook/react'
 
-storiesOf('Index Explorer', module)
+storiesOf('Components (Unthemed)|Index Explorer', module)
```
Comment (daedalus28) [inline stories/explorer/index.stories.js]: This should probably go under live demos

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>:exclamation: This PR is BIG (+4305 -4288)  
Please keep it below 500 net changes</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>You should add at least 1 more reviewer to the PR</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f68d2e4f648dadafb1a06459800cd965e70b4fa9
</p>

**Comment by daedalus28** [comment]
@stellarhoof crap, I think my attempt at avoiding conflicts just created more 😞 

Should be easy to fix though, sorry 😅

**Comment by daedalus28** [comment]
I'm fixing this for you, don't worry

**Pull request merged**